segmentation/include/pcl/segmentation/euclidean_cluster_comparator.h

```diff
@@ -194,71 +194,25 @@ namespace pcl
     *
     * \author Alex Trevor
     */
-  template<typename PointT, typename PointNT, typename PointLT = deprecated::T>
+  template<typename PointT, typename PointLT = deprecated::T>
   class EuclideanClusterComparator : public experimental::EuclideanClusterComparator<PointT, PointLT>
   {
     protected:
 
       using experimental::EuclideanClusterComparator<PointT, PointLT>::exclude_labels_;
 
     public:
-
-      using PointCloudN = pcl::PointCloud<PointNT>;
-      using PointCloudNPtr = typename PointCloudN::Ptr;
-      using PointCloudNConstPtr = typename PointCloudN::ConstPtr;
-
-      using Ptr = shared_ptr<EuclideanClusterComparator<PointT, PointNT, PointLT> >;
-      using ConstPtr = shared_ptr<const EuclideanClusterComparator<PointT, PointNT, PointLT> >;
+      using Ptr = shared_ptr<EuclideanClusterComparator<PointT, PointLT> >;
+      using ConstPtr = shared_ptr<const EuclideanClusterComparator<PointT, PointLT> >;
 
       using experimental::EuclideanClusterComparator<PointT, PointLT>::setExcludeLabels;
 
       /** \brief Default constructor for EuclideanClusterComparator. */
-      PCL_DEPRECATED(1, 12, "remove PointNT from template parameters")
```

Comment:
Is what I have done, the intended thing to do?

Reply:
At a glance, yes

Reply:
Not fully. You should delete the current `EuclideanClusterComparator` class entirely and move `experimental::EuclideanClusterComparator<PointT, PointLT>` ->  `EuclideanClusterComparator<PointT, PointLT>`